app/views/foods/index.html.erb

```diff
@@ -0,0 +1,35 @@
+
+<div class="conatiner-fluid shadow m-5 p-5">
```

Comment:
Well done on this. 

I just want to point out this class name on line 2 of this `app/views/foods/index.html.erb` file. Shouldn't it be container?

If not, then it is okay.

Reply:
Its personal preference , but its ok I will change it